docs/ingestion/native-batch.md

```diff
@@ -846,6 +847,15 @@ S3 Object:
 |bucket|Name of the S3 bucket|None|yes|
 |path|The path where data is located.|None|yes|
 
+Properties Object:
+
+|property|description|default|required?|
+|--------|-----------|-------|---------|
+|accessKeyId|S3 access key for this S3 InputSource|None|yes if secretAccessKey is given|
+|secretAccessKey|S3 secret key for this S3 InputSource|None|yes if accessKeyId is given|
+
+**Note :** *If any property is not given in the Properties Object, then the current default server value will be use.*
```

Comment:
```suggestion
Default values are used for any property not overridden in the Properties Object.
```
Caveat to my edit suggestion: it says "any property" but really it's just S3 access key that are supported, no? To avoid the premature generalization, could say something like: 

"If you don't provide an access key, the default S3 credentials provider chain is used." (With a link to the relevant section in s3.md).

Reply:
That is the idea for the future when we support more than just the access key and secret key. Will changed to "If you don't provide an access key, the default S3 credentials provider chain is used." for now and we can change it back in the future.

Reply:
done